doc/src/manual/functions.md

```diff
@@ -347,7 +347,7 @@ not work.
 
 It is often convenient to be able to write functions taking an arbitrary number of arguments.
 Such functions are traditionally known as "varargs" functions, which is short for "variable number
-of arguments". You can define a varargs function by following the last argument with an ellipsis:
+of arguments". You can define a varargs function by following the last non-keyword argument with an ellipsis:
```

Comment:
Keyword arguments haven't been introduced at this point in the docs, so I'd either omit this or say something like "the last positional argument"